src/Apps/Artist/Routes/AuctionResults/AuctionResultsFilterContext.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useContext, useReducer } from "react"
+
+export interface AuctionResultsFilters {
+  organizations?: string[]
+  openedItemIndex: number
+  page?: number
+  sort?: string
+}
+
+interface AuctionResultsFiltersState extends AuctionResultsFilters {
+  reset?: boolean
+}
+/**
+ * Initial filter state
+ */
+export const initialAuctionResultsFilterState: AuctionResultsFilters = {
+  organizations: [],
+  openedItemIndex: -1,
```

Comment:
#reallyminor instead of `-1` to represent no selected row for expansion, maybe it should be `null` (and than replace those `-1` comparisons with a more exist-y check). I guess the typing for that would have to change to `openedItemIndex: number | null` or something.